packages/runtime/container-runtime/src/runtimeLayerCompatState.ts

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	checkLayerCompatibility,
+	type ILayerCompatDetails,
+	type ILayerCompatSupportRequirements,
+} from "@fluid-internal/client-utils";
+import type { ICriticalContainerError } from "@fluidframework/container-definitions";
+import { UsageError } from "@fluidframework/telemetry-utils/internal";
+
+import { pkgVersion } from "./packageVersion.js";
+
+/**
+ * The core compatibility details of the Runtime layer that is the same across all layer boundaries.
+ * @internal
+ */
+export const RuntimeCoreCompatDetails = {
```

Comment:
I don't think this should be capitalized.  When I saw it imported elsewhere it confused me, my brain kept thinking it was a class or something.